### PR TITLE
Add Brew cask file for 0.6.5

### PR DESCRIPTION
**Pull request by thieman**
@jsingle cc @sporkmonger Seems to be working

### CODE REVIEW

File: setup/dusty.rb [hunk 1]
```diff
@@ -1,113 +1,38 @@
...
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
-  end
-
-  test do
-    Language::Python.each_python(build) do |python, _version|
-      system python, "-c", "import dusty"
-    end
-  end
+cask 'dusty' do
+  version '0.6.5'
```
Comment (jsingle) [inline setup/dusty.rb]: If we need to update this each release, can you update the release checklist in our docs?

**Comment by jsingle** [comment]
Sick, looks way better than the attempt to install it from source haha

File: setup/dusty.rb (hunk 1 above)
Comment (jsingle) [inline setup/dusty.rb]: Or maybe there's a way to have the release job replace it automatically?

Comment (thieman) [inline setup/dusty.rb]: Yeah have various doc updates to make still

**Comment by thieman** [comment]
Yeah after seeing your old file I definitely remember why we abandoned this path, didn't know about cask at the time (or at least I didn't)

**Pull request merged**